keyboards/willoucom/keypad/keymaps/via/keymap.json

```
@@ -0,0 +1,39 @@
+{
+    "keyboard": "willoucom/keypad",
```

move VIA_ENABLE to `keymap.json`
```suggestion
    "keyboard": "willoucom/keypad",
    "config": {
        "features": {
            "via": true
        }
    },
```